src/companion.rs

```diff
@@ -0,0 +1,136 @@
+use anyhow::Context;
+use regex::Regex;
+use tokio::process::Command;
+
+use crate::github_bot::GithubBot;
+
+pub async fn companion_update(
+	github_bot: &GithubBot,
+	owner: &str,
+	repo: &str,
+	branch: &str,
+) -> anyhow::Result<()> {
+	let token = github_bot.client.auth_key().await?;
+	Command::new("rustup")
+		.arg("update")
```

Comment:
We should not update rust. Can we not use the same docker image as we use for building substrate?

Reply:
The reason I update here is in case the bot is online for weeks/months.  You don't think it will be an issue?

Reply:
`rustup update` removed and we now use the same image as for substrate